packages/next/src/build/collect-build-traces.ts

```diff
@@ -0,0 +1,591 @@
+        config.experimental?.turbotrace?.contextDirectory ??
+        outputFileTracingRoot
+
+      // Under standalone mode, we need to trace the extra IPC server and
+      // worker files.
+      const isStandalone = config.output === 'standalone'
+      const nextServerEntry = require.resolve('next/dist/server/next-server')
+      const sharedEntriesSet = [
+        ...(config.experimental.turbotrace
+          ? []
+          : Object.keys(defaultOverrides).map((value) =>
+              require.resolve(value, {
+                paths: [require.resolve('next/dist/server/require-hook')],
+              })
+            )),
+        require.resolve('next/dist/compiled/next-server/app-page.runtime.prod'),
+        require.resolve(
+          'next/dist/compiled/next-server/app-route.runtime.prod'
+        ),
+        require.resolve('next/dist/compiled/next-server/pages.runtime.prod'),
+        require.resolve(
+          'next/dist/compiled/next-server/pages-api.runtime.prod'
+        ),
+      ]
+
+      const { incrementalCacheHandlerPath } = config.experimental
+
+      // ensure we trace any dependencies needed for custom
+      // incremental cache handler
+      if (incrementalCacheHandlerPath) {
+        sharedEntriesSet.push(
+          require.resolve(
+            path.isAbsolute(incrementalCacheHandlerPath)
+              ? incrementalCacheHandlerPath
+              : path.join(dir, incrementalCacheHandlerPath)
+          )
+        )
+      }
+
+      const serverEntries = [
+        ...sharedEntriesSet,
+        ...(isStandalone
+          ? [
+              require.resolve('next/dist/server/lib/start-server'),
+              require.resolve('next/dist/server/next'),
+              require.resolve('next/dist/server/require-hook'),
+            ]
+          : []),
+        require.resolve('next/dist/server/next-server'),
+      ].filter(Boolean) as string[]
+
+      const minimalServerEntries = [
+        ...sharedEntriesSet,
+        require.resolve('next/dist/compiled/next-server/server.runtime.prod'),
+      ].filter(Boolean)
+
+      const additionalIgnores = new Set<string>()
+
+      for (const glob of excludeGlobKeys) {
+        if (isMatch('next-server', glob)) {
+          outputFileTracingExcludes[glob].forEach((exclude) => {
+            additionalIgnores.add(exclude)
+          })
+        }
+      }
+      const ignores = [
+        '**/*.d.ts',
+        '**/*.map',
+        isStandalone ? null : '**/next/dist/compiled/jest-worker/**/*',
+        '**/next/dist/compiled/webpack/(bundle4|bundle5).js',
+        '**/node_modules/webpack5/**/*',
+        '**/next/dist/server/lib/squoosh/**/*.wasm',
+        '**/next/dist/server/lib/route-resolver*',
+        ...(ciEnvironment.hasNextSupport
+          ? [
+              // only ignore image-optimizer code when
+              // this is being handled outside of next-server
+              '**/next/dist/server/image-optimizer.js',
+              '**/node_modules/sharp/**/*',
```

Comment:
@ijjk I'm encountering an error after upgrading to 13.5.4 (the same code works fine on 13.5.3):

```
Error: Cannot find module 'sharp'
```

Also, after the upgrade, `node_modules/.pnpm/sharp@0.32.2` no longer appears in the traced files list.

I have a hunch that this issue might have something to do with the new code. Mind taking a look into it, please?

Reply:
Getting the same issue

Reply:
Hi, just to confirm are you using sharp in your own code? If so we will need to tweak this ignore as it's assuming we can ignore the Next.js' usage of sharp

Reply:
> Hi, just to confirm are you using sharp in your own code? If so we will need to tweak this ignore as it's assuming we can ignore the Next.js' usage of sharp 

I was using Plaiceholder. Seems it couldn't find sharp regardless of whether I had it installed as a dependency or not on Vercel. Worked fine locally.

Reply:
Thanks for the info should be fixed in https://github.com/vercel/next.js/pull/56674

Reply:
@isaackoz did the latest release fix your issue using Plaiceholder? I'm still getting the error with 13.5.3, 13.5.4 and 13.5.5.